app/src/main/java/swati4star/createpdf/fragment/FavouritesFragment.java

```diff
@@ -54,6 +56,7 @@
         implements SharedPreferences.OnSharedPreferenceChangeListener,
         View.OnClickListener {
     private SharedPreferences mSharedpreferences;
+    private boolean mDoesFavouritesExist = false;
```

Comment:
this assignment of `false` has no use right? As you are assigning again with same at line 155.

Reply:
Yes, this initialisation can be removed.

Reply:
Solved.